### PR TITLE
Back and submit button are the same size now, MGH logo added to sideb…

### PR DESCRIPTION
**Pull request by sandeepsuri**

### CODE REVIEW

File: emr/application/views/screeners/add.php [hunk 1]
```diff
@@ -106,12 +106,12 @@
 							</div>	
 							<div class="col-xs-6">
 								<ul class="nav navbar-right panel_toolbox">
-									<li><button type="button" class="btn btn-danger" onclick=history.back()>Back</button></li>
+									<li><button type="button" class="btn btn-danger" onclick=history.back() style="padding: 5px 18px 5px 18px; margin: 5px 0 5px 5px;">Back</button></li>
```
Comment (jas2996) [inline emr/application/views/screeners/add.php]: you shouldn't do history.back(), and you should have a href instead

File: emr/application/views/main/sidebar.php [hunk 2]
```diff
@@ -1,19 +1,17 @@
-  <body class="nav-md">
+<body class="nav-md">
     <div class="container body">
       <div class="main_container">
         <div class="col-md-3 left_col">
           <div class="left_col scroll-view">
             <div class="navbar nav_title" style="border: 0;">
-              <a href="main/index" class="site_title"><img src="../img/rsz_1certuslogowatermark.png"><span>Certus</span></a>            
+                <a href="main/index" class="site_title"><span><img src="../img/MGH.svg" style="padding: 0 0 10px 40px; "></span></a>
```
Comment (jas2996) [inline emr/application/views/main/sidebar.php]: I think it looks better with 50px instead of 40px

**Review by jas2996 — CHANGES_REQUESTED** [review]
See my comments below, also when the menu is collapsed or when viewing in mobile the logo is not there, maybe have just a MGH text when the menu is collapsed

File: emr/application/views/screeners/add.php (hunk 1 above)
Comment (sandeepsuri) [reply emr/application/views/screeners/add.php]: Changed

File: emr/application/views/main/sidebar.php (hunk 2 above)
Comment (sandeepsuri) [reply emr/application/views/main/sidebar.php]: Changed

File: emr/application/views/screeners/add.php [hunk 3]
```diff
@@ -106,7 +106,7 @@
 							</div>	
 							<div class="col-xs-6">
 								<ul class="nav navbar-right panel_toolbox">
-									<li><button type="button" class="btn btn-danger" onclick=history.back() style="padding: 5px 18px 5px 18px; margin: 5px 0 5px 5px;">Back</button></li>
+									<li><button type="button" class="btn btn-danger" onclick="window.location.href='screeners'" style="padding: 5px 18px 5px 18px; margin: 5px 0 5px 5px;">Back</button></li>
```
Comment (jas2996) [inline emr/application/views/screeners/add.php]: no not this way, just like href = the page you want to go, like how the sidebar is

File: emr/application/views/screeners/add.php (hunk 1 above)
Comment (jas2996) [reply emr/application/views/screeners/add.php]: refer to my new comment on your changed code

File: emr/application/views/main/sidebar.php (hunk 2 above)
Comment (jas2996) [reply emr/application/views/main/sidebar.php]: Looks good.

**Comment by jas2996** [comment]
you still haven't fixed this "when the menu is collapsed or when viewing in mobile the logo is not there"

File: emr/application/views/screeners/add.php (hunk 3 above)
Comment (sandeepsuri) [reply emr/application/views/screeners/add.php]: is the new push good?

Comment (jas2996) [reply emr/application/views/screeners/add.php]: yup!

**Comment by jas2996** [comment]
The logo when the menu is collapsed looks good but on `screeners/add` there is a white box above the form, the form needs to be within the white box

**Comment by sandeepsuri** [comment]
Screener form back in white box

**Comment by jas2996** [comment]
Looks good to go!

**Pull request merged**